src/main/java/net/atos/zac/app/taken/TakenRESTService.java

```diff
@@ -295,14 +296,14 @@ private void createDocuments(final RESTTaak restTaak) {
         }
     }
 
-    private void ondertekenEnkelvoudigInformatieObjecten(final Map<String, String> taakdata) {
+    private void ondertekenEnkelvoudigInformatieObjecten(final Map<String, String> taakdata, final Zaak taakZaak) {
         if (taakdata.containsKey(TAAK_ELEMENT_ONDERTEKENEN) && taakdata.get(TAAK_ELEMENT_ONDERTEKENEN) != null) {
```

Comment:
Het heeft niets met jou wijziging van doen maar hier worden een aantal onnodige operaties gedaan:
1) taakdata.containsKey(TAAK_ELEMENT_ONDERTEKENEN)
2) taakdata.get(TAAK_ELEMENT_ONDERTEKENEN)
3) taakdata.get(TAAK_ELEMENT_ONDERTEKENEN)

Als voor de if nu een 
final String ondertekenen = taakdata.get(TAAK_ELEMENT_ONDERTEKENEN); 
wordt gedaan dan kan deze gebruik worden in de if (de get() geeft null terug als de key er niet is) en daarna voor de split

Reply:
Helder - dit is dus ook zo'n geval van vaker dan nodig dezelfde informatie ophalen. Die staat inmiddels op mijn controle-lijst!

Opgelost in opvolgende commit